applications/connecting_applications_to_services/projecting-binding-data.adoc

```diff
@@ -0,0 +1,23 @@
+[id="projecting-binding-data"]
+= Projecting binding data
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: projecting-binding-data
+
+toc::[]
+
+This section provides information on how you can consume the binding data.
+
+== Consumption of binding data
+For accessing and consuming the binding data from a workload, this data has to be projected into the workload from a backing service. After the backing service exposes the binding data, {servicebinding-title} Operator automatically projects this set of data into the workload in the following methods:
+
+. By default, as files.
```

Comment:
```suggestion
```

Reply:
As I have said earlier, SBO is offering these two methods from which users can choose and we need to state it explicitly.